docs/components/radio.md

```diff
@@ -171,6 +171,98 @@ const cardRadio = ref<string>('')
 </script>
 ```
 
+### cardOrientation
+
+Prop for changing card orientation. Default value is `vertical`, the alternative to which is `horizontal`.
+
+<KRadio
+  v-model="horizontalCard"
+  card
+  card-orientation="horizontal"
+  description="Choose this option if you want your APIs to only be accessible from within your private network."
+  label="Private"
+  selected-value="private"
+>
+  <KBadge appearance="success">
+    Recommended
+  </KBadge>
+</KRadio>
+<KRadio
+  v-model="horizontalCard"
+  card
+  card-orientation="horizontal"
+  description="Choose this option if you want your APIs to be publicly accessible by anyone on the internet."
+  label="Public"
+  selected-value="public"
+/>
```

Comment:
_(I'm not sure anything can be done here, just pointing it out)_

Using this docs layout as an example, I'm not sure it looks great that the text between the two horizontal cards doesn't line up when the Recommended badge is present
![image](https://github.com/user-attachments/assets/a0a853c8-e733-4fc6-91bc-74c77f29ef03)

Notice the text in the second card extends further to the right. If I make the text longer it's more noticable:

![image](https://github.com/user-attachments/assets/51913efd-7b08-49d2-aee4-dd325a8f78b7)